ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-provision-chain.yaml

```diff
@@ -0,0 +1,29 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-provision
+  steps:
+    - ref: ipi-install-rbac
+    - ref: openshift-cluster-bot-rbac
+    - ref: ipi-conf
```

Comment:
As cluster name is "${NAMESPACE}-${JOB_NAME_HASH}", suggest to move the step "ipi-conf" forward several steps, e.g. rightly before the step "ipi-conf-mirror".